glossary.md

```diff
@@ -5,7 +5,9 @@ Arduinos are usually only loaded with only one program. See https://en.wikipedia
 
 Clone: To copy a remote repository to a computer. See https://help.github.com/articles/cloning-a-repository/. Also see Repository. 
 
-GitHub: A repository-hosting website that we use in this project. See https://en.wikipedia.org/wiki/GitHub. Also see Repository.
+Distributor: The device that will route our input modules to our output modules. Right now, we think that the distributor may end up being a Raspberry Pi. See  https://github.com/jjthrash/room-of-sound/blob/master/room-of-sound.md. Also see Room of Sound. 
```

Comment:
@kerbin111 For what it's worth I think relative links work. So you can link to, e.g. `[the spec](room-of-sound.md)`